interfaces/acados_template/acados_template/c_templates/template_example.in.c

```diff
@@ -0,0 +1,385 @@
+/*
+ *    This file is part of acados.
+ *
+ *    acados is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation; either
+ *    version 3 of the License, or (at your option) any later version.
+ *
+ *    acados is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Lesser General Public
+ *    License along with acados; if not, write to the Free Software Foundation,
+ *    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#include <stdio.h>
+
+#include <acados/utils/print.h>
+#include <acados/ocp_qp/ocp_qp_partial_condensing_solver.h>
```

Comment:
Indeed. Unfortunately I might not have time to switch to the new ocp_nlp C before January or so, since both for the project at Bosch and in Munich we urgently need a way to set up ocp_nlp solvers. But after this is done we should definitely update the templates ;)